src/styles/components/service-table/styles.less

```diff
@@ -34,7 +34,7 @@
       }
 
       &-status {
```

Comment:
These changes make sense, but there will be conflicts when @GeorgiSTodorov merges the service table replacement PR.

I don't like how much content we're trying to squeeze into this table, but I trust we've tested this with users and this is the data they absolutely need to see in this view.

Reply:
In theory we could drop these changes if the service PR has merged but also we can drop all the stylings of the columns made in this file.

Reply:
My PR has two reviews, but CI is broken.